backend/src/utils/stream.py

```
@@ -155,12 +155,11 @@ def handle_multi_mode(chunk: dict):
         if "messages" in chunk:
             i0, i1 = chunk[0], chunk[1]
             msg = i1[0]
-            
-            if agent_name := dict(msg).get("lc_agent_name"):  
-                if agent_name != current_agent:  
-                    logger.warning(f"🤖 {agent_name}: ")  
-                    current_agent = agent_name  
 
+            if agent_name := dict(msg).get("lc_agent_name"):
+                if agent_name != current_agent:
```

Ruff (0.14.13)

Undefined name `current_agent`

(F821)